src/swagger/trips.swagger.js

```diff
@@ -436,3 +440,59 @@
 *       '404':
 *             description: request not found.
 * */
+
+/**
+ * @swagger
+ * /api/v1/trips/{requestId}/view:
+ *   get:
+ *     security:
+ *       - bearerAuth: []
+ *     tags:
+ *       - Trips
+ *     name: User view a trip request
+ *     summary:  User should be able to view a trip request he made
```

Comment:
doesn't the manager also view a request made by someone in their direct report

Reply:
I added the description